src/chains.ts

```diff
@@ -43,6 +43,15 @@ export interface Chain {
     fee_collector?: string;
     staking?: StakingContract[];
   }
+  theGraph?: {
+    tokens_whitelist: string[];
```

Comment:
@iam-robi from my understanding the new Subgraph dynamically picks up the whitelisted tokens. So is this then still necessary?

Also the Chain interface is getting pretty verbose. I think it makes more sense to have another interface for the Graph which then is included in the Chain interface

Reply:
In the new script, the Subgraph's settings files are set up with the SDK https://github.com/wagmiog/exchange-contracts/blob/6b5cf3c5f5f84d4f4afa1a02aa7e4e68ff1c48c2/scripts/deploy-graph.js#L42
We  unfortunately can't get the token adresses' dynamically on each chain, and which tokens we want to be whitelisted on each chain. So we need to write the addresses on the SDK beforehand
The whitelist is only to calculate a token price.
Yes it makes more sense, i will do another interface for the Graph.